watchtower/blob/commitments.go

```diff
@@ -151,3 +151,38 @@ func (c CommitmentType) ParseRawSig(witness wire.TxWitness) (lnwire.Sig,
 			c)
 	}
 }
+
+// NewJusticeKit can be used to construct a new JusticeKit depending on the
+// CommitmentType.
+func (c CommitmentType) NewJusticeKit(sweepScript []byte,
+	breachInfo *lnwallet.BreachRetribution, withToRemote bool) (JusticeKit,
```

Comment:
Any reason this actually needs to return the interface? This forces any callers to cast if they want to use the underlying justice kit. If we return the concrete type, then it can still be used anywhere that `JusticeKit` is accepted.

Reply:
Same below re the empty kit.